pipelines.adoc

```diff
@@ -1,266 +1,189 @@
+
+Alternatively, if you are using an unmanaged cluster, you can set up a persistent volume to be used by the pipeline. A simple example pv definition is provided in the following example. Update the path and other values in your `pv.yaml` file to suit your requirements. For instance, NFS is the preferred configuration in a production environment, rather than using local storage, as shown in the example.
+
+. Log in to your cluster. For example, for OKD:
+
+```
 oc login <master node IP>:8443
-....
+```
 
-Clone the pv.yaml in this repo and apply it.
+. Clone the `pv.yaml` file in this repo and apply it.
 
-....
+```
 cd ./pipelines/common
 oc apply -f pv.yaml -n kabanero
-....
+```
 
-==== Create secrets to pull from git repo and push to docker registry
+==== Create secrets
 
-This has to be created in the `+kabanero+` namespace and associated with
-the service account that pipelines are run with. The secrets can be
-created in a few different ways. Simplest option is to go configure this
-is the Tekton Dashboard under the secrets section as given
-https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md#create-credentials-git[here].
+Git secrets must be created in the `+kabanero+` namespace and associated with the service account that runs the pipelines. To configure secrets by using the Tekton Dashboard, see
+link:https://kabanero.io/docs/ref/general/tekton-webhooks.html#create-secrets[Create secrets].
 
-Alternatively you can configure this in the OKD console or you can setup
-the secret using the OKD CLI.
+Alternatively, you can configure secrets in the OKD console or you can set up the secret by using the OKD CLI.
 
-The following video that walks through setting up the prereqs to run the
-pipelines: https://youtu.be/MfS05SU9yIM
 
-=== Pipeline execution using Tekton Dashboard Webhook Extension
 
-You can leverage the Tekton Dashboard Webhook Extensions to drive the
-pipelines automatically by configuring webhooks to github. This enables
-the CI/CD flow by automatically building and deploying the applications
-with code updates in your git repo. Events such as commits or pull
-requests in a github repo can be setup to automatically trigger pipeline
-runs.
+=== Execute pipelines by using the Tekton Dashboard Webhook Extension
 
-For detailed description on creating a web hook to git, please visit
-these resources:
-https://kabanero.io/docs/ref/general/tekton-webhooks.html
-https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md
+You can link:https://kabanero.io/docs/ref/general/tekton-webhooks.html[use the Tekton Dashboard Webhook Extension to drive pipelines] that automatically build and deploy an application whenever you update the code in your Git repo. Events such as commits or pull requests can be set up to automatically trigger pipeline runs.
```

Comment:
Maybe add for a more detailed guide on setting up a webhook to drive a kabanero pipeline follow these instructions:  https://github.com/tektoncd/experimental/blob/master/webhooks-extension/docs/GettingStarted.md